server/models/account/user.ts

```diff
@@ -336,11 +374,16 @@ export class UserModel extends Model<UserModel> {
   }
 
   isAbleToUploadVideo (videoFile: { size: number }) {
-    if (this.videoQuota === -1) return Promise.resolve(true)
+      if (this.videoQuota === -1 && this.videoQuotaDaily === -1) 
```

Comment:
Keep the return on the same line than `if`

Reply:
That makes a pretty long line, but okay.